src/components/contact/Contact.js

```diff
@@ -24,14 +22,17 @@ const Contact = ({ t }) => {
         <img src={peopleSearch} alt="people search logo" />
       </div>
       <div className={styles.rightSection}>
-        <h1>{t('contact-page-header')}</h1>
+        <h1>
+          <Trans id="contact-page-header" />
```

Comment:
I think we should always set a default value: `<Trans id="contact.header" >Contact</Trans>`